ftw/testing/freezer.py

```diff
@@ -13,12 +13,12 @@ def __init__(self, new_now):
         self.new_now = new_now
 
     def forward(self, **kwargs):
-        self.new_now = datetime.now() + timedelta(**kwargs)
+        self.new_now += timedelta(**kwargs)
```

Comment:
Datetime arithmetic (`+=`) should only ever be performed on non-naive objects:

> In addition, if you perform date arithmetic on local times that cross DST boundaries, the result may be in an incorrect timezone (ie. subtract 1 minute from 2002-10-27 1:00 EST and you get 2002-10-27 0:59 EST instead of the correct 2002-10-27 1:59 EDT). A normalize() method is provided to correct this.
http://pytz.sourceforge.net/

But we don't know the TZ for a naive object here, this would make calling `timezone.normalize(result_dt)` really cumbersome. So 👍  for leaving it this way for now, with the argument that people that pass naive datetimes into freezer don't deserve any better 😜